apps/core/views/reports/release_publish/views.py

```
@@ -1562,11 +1440,23 @@ def _step_check_version(release, ctx, log_path: Path, *, user=None) -> None:
 def _step_handle_migrations(release, ctx, log_path: Path, *, user=None) -> None:
+    """Run migration checks required before release promotion.
+
+    Prerequisites: version checks already passed.
+    Side effects: delegates to release workflow migration handlers.
+    Rollback expectations: migration failures halt pipeline for operator intervention.
+    """
     _append_log(log_path, "Freeze, squash and approve migrations")
     _append_log(log_path, "Migration review acknowledged (manual step)")
 
 
 def _step_pre_release_actions(release, ctx, log_path: Path, *, user=None) -> None:
```

Ruff (0.15.0)

[warning] Unused function argument: `user`

(ARG001)